include/glow/Runtime/StatsExporter.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_RUNTIME_STATSEXPORTER_H
+#define GLOW_RUNTIME_STATSEXPORTER_H
+
+#include "llvm/ADT/StringRef.h"
+
+namespace glow {
+
+/// Interface for exporting runtime statistics.  The base implementation
+/// delegates to any subclass registered via `registerStatsExporter`.
+class StatsExporter {
```

Comment:
Yeah this matches internal terminology. This stub doesn't do any actual exporting (subclasses will) but I think thats the interface still, so I like it.

Reply:
Yeah on the naming (this and `incrementCounter`) I'm pretty much following fb303 terminology.  Obviously that's not strictly necessary, I just figured it would be more obvious to FB internal users.